msdos/makefile

```diff
@@ -83,7 +83,7 @@ hc386.arg:  msdos\Makefile
               -Hnocopyr
               -Hpragma=Offwarn(491,553,572)
               -Hon=Recognize_library  # make memcpy/strlen etc. inline
-              -Hoff=Behaved           # turn off some optimiser warnings
+              -Hoff=Behaved           # turn off some optimizer warnings
```

Comment:
That's British (and Irish and Australian and New Zealand?) vs. American (and Canadian?) spelling, and there's no {{Use American spelling}} template on libpcap, tcpdump, or tcpslice :-), so I don't know whether we should leave that one alone or not. (And British spelling may commonly be used for English in continental Europe and former British colonies other than the ones mentioned.)

Reply:
Yeah, I'm aware. I looked for other things and found a bunch of other instances of optimize: https://github.com/search?q=repo%3Athe-tcpdump-group%2Flibpcap+optimize&type=code

I'm happy to drop it if it's controversial, but as it was a lonely instance vs a preponderance, it seemed worth suggesting.

Reply:
I way prefer British spelling, but that will never win, so I don't care.

Reply:
If there are no objections, I would prefer British -- maybe this would help me to remember better what it is.

Reply:
I'm OK with British, US, or other Anglospheric spellings.

Reply:
grep optimize:117
grep optimise:1
For consistency I am OK with the change.

Reply:
> If there are no objections, I would prefer British

It should probably be consistent; I'm OK with either spelling, but most of the spelling is US-style.